### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by dguido**
- Bump all the requirements and installation instructions to the bottom
- Eliminate the statement about Manticore being beta software
- Add more background material to the Ethereum section
- Rename Quickstart to Installation and Installation to Expert Installation

---
This change is [Reviewable](https://reviewable.io/reviews/trailofbits/manticore/1037)

### CODE REVIEW

**Comment by mossberg** [comment]
This is more about a266c0beeb2b5f0586bd002d6d0814bce0655289 than this PR but I think merging the Quick Start and Installation sections has made things a little messy. imo I think we should keep them as separate sections.

specifically:

- i'm not sure it makes sense to have commands for downloading, building, and running manticore on examples in the "installation" section, as those are not directly related to installation of manticore
- there's now redundancy between Option 4 in "Expert Installation" and the Docker install section
-  I'm not a huge fan of the "Expert Installation" language. I feel like it's a slight insult to the intelligence of the readers/users :P

File: README.md [hunk 1]
```diff
@@ -65,15 +60,17 @@ m.run()
 
 ### Ethereum
 
-Manticore ships with _detectors_ accessible from the command line. Solidity smart contracts must have a `.sol` extension to be consumed by Manticore.
+Manticore includes a symbolic Ethereum Virtual Machine (EVM) and a convenient interface for automated compilation and analysis of Solidity. It also integrates with [Ethersplay](https://github.com/trailofbits/ethersplay), Trail of Bits’ visual disassembler for EVM bytecode, for analysis visualization. As with binaries, Manticore offers a simple command line interface and a Python API for analysis of EVM bytecode. See a demo: https://asciinema.org/a/154012
+
+Use the CLI to explore possible states in Ethereum smart contracts. Manticore includes _detectors_ which flag certain conditions, including known vulnerable code, as it explores possible states. Note: Solidity smart contracts must have a `.sol` extension for consumption by Manticore.
```
Comment (mossberg) [inline README.md]: i didn't get a chance to comment on the actual pr that introduced it, but i think "consumption" is a slightly weird word to use here, imo "analysis" would be a better fit

**Comment by dguido** [comment]
@mossberg can you add the duplicated instructions to the Docker quickstart rather than saying "Then follow from the make command above."?

also, I think this note "Note: If you are experiencing unanticipated errors when running Manticore on native binaries, you can try using the --process-dependency-links pip flag. This will install the development branch of our disassembler dependency, which may contain useful bug fixes." is probably a better fit for the FAQ than the main readme.

**Comment by mossberg** [comment]
@dguido done.

**Comment by dguido** [comment]
I think this is in a decent place now. Make any changes you want and then merge it!

**Pull request merged**